alicloud/service_alicloud_cms.go

```diff
@@ -69,3 +71,16 @@ func (s *CmsService) WaitForCmsAlarm(id string, enabled bool, timeout int) error
 	}
 	return nil
 }
+
+func (s *CmsService) BuildJsonWebhook(webhook string) string {
+	return fmt.Sprintf("{\"method\":\"post\",\"url\":\"%s\"}", webhook)
+}
+
+func (s *CmsService) ExtractWebhookFromJson(webhookJson string) (string, error) {
+	byt := []byte(webhookJson)
+	var dat map[string]interface{}
+	if err := json.Unmarshal(byt, &dat); err != nil {
+		return "", err
+	}
+	return dat["url"].(string), nil
+}
```

Comment:
If there is a new parameter, please add it in the resource docs.

Reply:
done